CONTRIBUTORS.txt

```diff
@@ -243,5 +243,8 @@ In chronological order:
   * Fix ``util.selectors._fileobj_to_fd`` to accept ``long``.
   * Update appveyor tox setup to use the 64bit python.
 
+* Akamai (through Jesse Shapiro) <jshapiro@akamai.com>
```

Comment:
TBH I'd emphasize Jesse Shapiro over Akamai here. Maybe something like:

```
* Jesse Shapiro <jshapiro@akamai.com>, work sponsored by Akamai
```

Reply:
In this case, I'm already in the list; this is specifically because the license points to this file as a list of copyright holders, and for my contributions on Akamai time, they're the copyright holder, not me.

Reply:
Hm, with that in mind, do we need to obtain anything from them that says they are not going to change their mind and try to sue Andrey or the rest of us over the work you've done?

Reply:
No more than we do for anyone else. In general, the copyright for the individual pieces of urllib3 are held by the original authors anyway; they're just (irrevocably) licensed under the MIT license.

I've gone through the internal process Akamai requires to allow contributions where Akamai retains the copyright; if we need a copyright assignment, then that's a matter of shifting the overall policy of the project. To be clear, I'd be fine with that; I'd just have to go through an additional process within Akamai to get that assignment set up; in addition, we'd need to set up a process to obtain that from everyone and a legal entity to actually hold the copyright for the project.

Reply:
@sigmavirus24, did the above address your concerns?

Reply:
Definitely. Sorry. The notification got lost :)